suzieq/db/parquet/parquetdb.py

```diff
@@ -651,45 +651,65 @@ def _get_filtered_fileset(self, dataset: ds, namespace: list) -> ds:
         Returns:
             ds: pyarrow dataset of only the files that match filter
         """
-        if not namespace:
+        def check_ns_conds(ns_to_test: str, filter_list: List,
+                           cond: operator.or_) -> bool:
+            # pylint: disable=comparison-with-callable
+            res = not cond == operator.or_
```

Comment:
What happens in this line? Also I don't understand why cond has that type hint

Reply:
Yes, you are right maybe a comment or making it more explicit would help.
If I want to concat the expression with a OR i want the initial value to be False
while with AND I want the initial value to be True